pkg/util/platform_status.go

```diff
@@ -27,6 +28,8 @@ func GetPlatformStatus(client client.Client, infra *configv1.Infrastructure) (*c
 		if status.AWS != nil && len(status.AWS.Region) > 0 {
 			return status, nil
 		}
+	} else {
```

Comment:
This condition introduces new behavior that wasn't there before.  Why is this now an error, when it wasn't before?

Reply:
We don't need it anymore. I deleted it.